examples/validate/check_hostpath.yaml

```diff
@@ -9,10 +9,10 @@ spec:
       kinds:
       - Pod
     validate:
-      message: "Host path should be /var/log"
+      message: "Host path is not allowed"
       pattern:
         spec:
           volumes:
-          - (name): log
+          - name: "*"
             hostPath:
-              path: /var/log
+              path: ""
```

Comment:
path: "" 
will require existing empty path string
currently, we do not have a feature that checks of absence of the parameter

Reply:
@tony-apr - we discussed and decided to support null and empty "" in the same manner to make this flexible for the user. See: https://github.com/nirmata/kyverno/issues/97